CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.8.2
+### Fixed
+- `Deserialization of profiles created in UpdateRepresentation`
```

Comment:
Please pull this out into an issue, and link the issue here. This makes it possible to track the issue in the future.